.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Runs a single command using the runners shell
+      - name: Run a one-line script
+        run: echo Hello, world!
+
+      # Runs a set of commands using the runners shell
+      - name: Run a multi-line script
+        run: |
+          echo Add other actions to build,
+          echo test, and deploy your project.
```

Comment:
- [ ] [REQUIRED] You install another linter, please implement [correct linters](https://github.com/microverseinc/linters-config/tree/master/html-css) and all linters should pass.

- Your result ❌ 

![image](https://user-images.githubusercontent.com/63915024/185154346-9de4705b-74cb-4980-b7d4-223f98db3ef5.png)

- It should be ✔️ 

![image](https://user-images.githubusercontent.com/63915024/185154527-aebb58d0-1878-4b54-936e-fedfc34437f0.png)